app/javascript/components/Articles/ArticlesDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'; 
+
+export interface ArticleDetailsProps {
+  title:string,
+  content:string,
+  user:string,
+  created_at:string
+}
+ 
+const ArticleDetails: React.SFC<ArticleDetailsProps> = (props) => {
```

Comment:
I think `React.SFC` is technically deprecated, so might be better to use just `React.FC`. Maybe look at [this StackOverflow post](https://stackoverflow.com/questions/53885993/react-16-7-react-sfc-is-now-deprecated)

Reply:
Interesting reads, thanks for pointing that out!